pymc/bart/bart.py

```diff
@@ -132,6 +137,13 @@ def logp(x, *inputs):
         """
         return at.zeros_like(x)
 
+    @classmethod
+    def get_moment(cls, rv, size, *rv_inputs):
```

Comment:
Could BART shape be determined by the shape of the rv_inputs? Or does the mean already integrate this info?

Reply:
The shape of BART is always the shape of observed data Y.

Reply:
And do you do anything with the size variable or is that branch below unnecessary?

Reply:
everything is a lie :-) Initially I tough about having a size argument for future extension. But probably that will not be very useful.